src/goto-analyzer/unreachable_instructions.cpp

```diff
@@ -167,7 +167,7 @@ void unreachable_instructions(
 {
   json_arrayt json_result;
 
-  std::set<irep_idt> called=
+  std::unordered_set<irep_idt> called=
```

Comment:
Nit pick: introduce a space before `=` to converge towards clang-format's suggested style.